File: src/main/java/site/gachontable/gachontablebe/domain/admin/usecase/ManagePub.java
package site.gachontable.gachontablebe.domain.admin.usecase;

import site.gachontable.gachontablebe.domain.auth.domain.AuthDetails;
import site.gachontable.gachontablebe.domain.admin.presentation.dto.request.PubManageRequest;

public interface ManagePub {

    String execute(AuthDetails authDetails, PubManageRequest request);
}

File: src/main/java/site/gachontable/domain/admin/port/in/ManagePub.kt
package site.gachontable.domain.admin.port.in;

import site.gachontable.infra.security.principal.AuthDetails;
import site.gachontable.presentation.admin.dto.request.PubManageRequest;

public interface ManagePub {

    String execute(AuthDetails authDetails, PubManageRequest request);
}
